### ISSUE TITLE
EGDCL: An Adaptive Curriculum Learning Framework for Unbiased Glaucoma Diagnosis

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#129]
# TL;DR
Glaucoma data has been a difficult task due to the presence of hard samples and data imbalance. They proposed a curriculum-learning EGDCL to learn it through a network that generates sample difficulty, label rarity, and feature map importance. Results beyond previous research.

![Input sample (Sec 3 1 Student Network](https://user-images.githubusercontent.com/24524018/95158767-68a5ee00-07d7-11eb-8ae9-a9b8429862e0.png)

# Why it matters:

# Paper URL
https://www.ecva.net/papers/eccv_2020/papers_ECCV/html/3685_ECCV_2020_paper.php

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Rongchang Zhao, Xuanlin Chen, Zailiang Chen, Shuo Li
- Central South University,
- Western University

# Methods

# Results

# Comments
